File: Admin/Models/SysUser.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Admin.Models
{
    public class SysUser:UserModel
    {
        public string File { get; set; }
        public bool Gender { get; set; }
        public bool isProfitA { get; set; }
        public bool isProfitB { get; set; }
        public bool isProfitC { get; set; }
    }
}

File: Admin/Scripts/PassiveModel/SysUser.js
'use strict';
class SysUser extends UserModel {
    constructor(file, gender, isProfitA, isProfitB, isProfitC) {
        super(id, firstName, mI, lastName, loginName, password, cell, website, email);
        this.File = file;
        this.IsProfitA = isProfitA;
        this.IsProfitB = isProfitB;
        this.IsProfitC = isProfitC;
    }
}